### PR TITLE
【Hackathon 5th No.41】为 Paddle 新增 Rprop API 中文文档

### PR DESCRIPTION
**Pull request by WintersMontagne10335**
为 Paddle 新增 Rprop API 中文文档

- https://github.com/PaddlePaddle/Paddle/pull/58851
- https://github.com/PaddlePaddle/Paddle/issues/57262

### CODE REVIEW

File: docs/api/paddle/optimizer/Rprop_cn.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
+.. _cn_api_paddle_optimizer_Rprop:
+
+Rprop
+-------------------------------
+
+.. py:class:: paddle.optimizer.Rprop(learning_rate=0.001, learning_rate_range=(1e-5, 50), parameters=None, etas=(0.5, 1.2), grad_clip=None, name=None)
+
+    
+.. note::
+此优化器仅适用于 full-batch 训练。
```
Comment (sunzhongkai588) [inline docs/api/paddle/optimizer/Rprop_cn.rst]:
```suggestion
    此优化器仅适用于 full-batch 训练。
```

File: docs/api/paddle/optimizer/Rprop_cn.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    \end{aligned}
+
+
+参数
+::::::::::::
+
+    - **learning_rate** (float|_LRScheduleri，可选) - 初始学习率，用于参数更新的计算。可以是一个浮点型值或者一个_LRScheduler 类，默认值为 0.001。
+    - **learning_rate_range** (tuple，可选) - 学习率的范围。学习率不能小于元组的第一个元素；学习率不能大于元组的第二个元素。
+    - **parameters** (list，可选) - 指定优化器需要优化的参数。在动态图模式下必须提供该参数；在静态图模式下默认值为 None，这时所有的参数都将被优化。
+    - **etas** (tuple，可选) - 用于更新学习率的元组。元组的第一个元素是乘法递减因子；元组的第二个元素是乘法增加因子。
```
Comment (sunzhongkai588) [inline docs/api/paddle/optimizer/Rprop_cn.rst]: 默认值都写一下

File: docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.Rprop.md
Comment (sunzhongkai588) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.Rprop.md:1]: @zhwesky2010 确认一下

**Review by zhwesky2010 — APPROVED** [review]
LGTM for api_difference/torch.optim.Rprop.md

File: docs/api/paddle/optimizer/Rprop_cn.rst (hunk 2 above)
Comment (WintersMontagne10335) [reply docs/api/paddle/optimizer/Rprop_cn.rst]: Done

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**